### ISSUE TITLE
BUG: SQL Operations Studio doesn't save result to Excel

### ISSUE THREAD

**Issue by JohnyL** [Microsoft/sqlopsstudio#2600]
- SQL Operations Studio Version: 0.32.8
- Operating system: Windows 10 Home x64

SQL Operations Studio doesn't save result to Excel. When I hit "Save", no file appears and no message pops up to indicate that file is created.  I could manage to save file once per session. Now I tried 20 times to save file - and all is in vain. 😐 Can this bug be connected with [previously fixed bug](https://github.com/Microsoft/sqlopsstudio/issues/1748)? 🤔

Steps to Reproduce:
1. Fire up new instance of SQL Operations Studio.
2. Type in `SELECT 1` and execute it.
3. Save result to Excel.

**Comment by ttilberg** [comment]
@MattIrv I came across this after getting set up on Ubuntu Desktop 18.04 using

```
Version 0.32.8
Commit efd470b3c771ef7473ba3cf108a33b6579c96a53
Date 2018-08-30T19:33:15.959Z
VS Code 1.23.1
Shell 1.7.16
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```

I thought I had tried everything and couldn't get an export (to csv) work. The note you gave about explicitly right click > Export (as opposed to select all, and still trying to use the export buttons) worked for me just now.

**Comment by JohnyL** [comment]
@MattIrv Yes, this workaround *does* work. Thanks!

**Comment by kburtram** [comment]
This should be fixed in the `0.32.9` release.  Please reactive if the problem persists.
